tidy3d/material_library/parametric_materials.py

```diff
@@ -0,0 +1,266 @@
+        intraband = self.drude_intraband
+        if self.include_interband:
+            interband = self.pole_residue_interband
+            intraband_poles = intraband.pole_residue.poles
+            interband_poles = interband.pole_residue.poles
+            poles = intraband_poles + interband_poles
+            pole_residue = PoleResidue(poles=poles)
+            return Medium2D(ss=pole_residue, tt=pole_residue)
+        return Medium2D(ss=intraband, tt=intraband)
+
+    @property
+    def drude_intraband(self) -> Drude:
+        """A Drude-type model for the intraband term of graphene.
+
+        Returns
+        -------
+        :class:`Drude`
+            A Drude-type model for the intraband term of graphene.
+
+        """
+        intra_f1 = np.sqrt(
+            (Q_e * K_B * self.temp / (HBAR**2 * 4 * np.pi**3 * EPSILON_0))
+            * (
+                self.mu_c / (K_B * self.temp)
+                + 2 * np.log(np.exp(-self.mu_c / (K_B * self.temp)) + 1)
+            )
+        )
+        intra_delta1 = (1 / HBAR) * self.gamma / np.pi
+        return Drude(coeffs=[(intra_f1, intra_delta1)])
+
+    @property
+    def pole_residue_interband(self) -> PoleResidue:
+        """A pole-residue model for the interband term of graphene. Note that this does not
+        include the intraband term, which is added in separately.
+
+        Returns
+        -------
+        :class:`.PoleResidue`
+            A pole-residue model for the interband term of graphene.
+        """
+
+        mu_c_freq = self.mu_c / (np.pi * HBAR)
+        temp_freq = K_B * self.temp / (np.pi * HBAR)
+
+        freq1 = mu_c_freq
+        freq2 = mu_c_freq + 5 * temp_freq
+
+        ind1 = int(
+            NUM_FREQS * (freq1 - MIN_FREQ_THZ * 1e12) / (MAX_FREQ_THZ * 1e12 - MIN_FREQ_THZ * 1e12)
+        )
+        ind2 = int(
+            NUM_FREQS * (freq2 - MIN_FREQ_THZ * 1e12) / (MAX_FREQ_THZ * 1e12 - MIN_FREQ_THZ * 1e12)
+        )
+        inds = np.array([[0, -1], [ind1, ind2]])
+
+        omega = 2 * np.pi * np.linspace(MIN_FREQ_THZ * 1e12, MAX_FREQ_THZ * 1e12, NUM_FREQS)
+        sigma_inter = self.interband_conductivity(omega)
+
+        return self._fit_interband_conductivity(omega, sigma_inter, inds)
+
+    def interband_conductivity(self, omegas: List[float]) -> List[complex]:
+        """Numerically integrate interband term."""
+
+        def fermi(E: float) -> float:
+            """Fermi distribution."""
+            return 1 / (np.exp((E - self.mu_c) / (K_B * self.temp)) + 1)
+
+        def fermi_g(E: float) -> float:
+            """Difference of fermi distributions."""
+            return fermi(-E) - fermi(E)
+
+        inter = np.zeros(len(omegas), dtype=complex)
+
+        def integrand(E, omega):
+            return (fermi_g(E * HBAR) - fermi_g(HBAR * omega / 2)) / (omega**2 - 4 * E**2)
+
+        for i, omega in enumerate(omegas):
+
+            integral, _ = integrate.quad(integrand, 1e11, 1e16, args=(omega,), epsabs=1e-20)
```

Comment:
what is 1e11 and 1e16? either add as kwargs or define as constants in the module. similar comment for epsabs.